food-truck-frontend/src/api/RateReviewApi.ts

```diff
@@ -11,37 +13,33 @@ import {ClientResponseAction} from "./ResponseTypes";
 export const loadReviewsByUser = async (
   userId: number,
   onFail?: ClientResponseAction
-) => {
-  let reviews: Review[] = [];
+): Promise<Review[] | null> => {
```

Comment:
If it returns async, it implicitly returns a promise of its specified return type. So I'd expect this to return `Promise<Promise<Review[] | null>>`. But I could be wrong.

Reply:
After some research, I've concluded this is handled correctly internally, no matter how many promises wrap each other. Please disregard this comment.